sidebars.js

```diff
@@ -1,376 +1,372 @@
-              label: '🧰 Installation',
-              items: [
-                'guides/setup/installation/local',
-                'guides/setup/installation/databricks',
-                'guides/setup/installation/spark_emr'
-              ]
-            },
-            {
-              type: 'category',
-              label: 'Configuring Data Contexts',
-              items: [
-                'guides/setup/configuring_data_contexts/how_to_create_a_new_data_context_with_the_cli',
-                'guides/setup/configuring_data_contexts/how_to_configure_datacontext_components_using_test_yaml_config',
-                'guides/setup/configuring_data_contexts/how_to_configure_credentials_using_a_yaml_file_or_environment_variables',
-                'guides/setup/configuring_data_contexts/how_to_configure_credentials_using_a_secrets_store',
-                'guides/setup/configuring_data_contexts/how_to_instantiate_a_data_context_without_a_yml_file'
-              ]
-            },
-            {
-              type: 'category',
-              label: 'Configuring metadata Stores',
-              items: [
-                'guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_amazon_s3',
-                'guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_azure_blob_storage',
-                'guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs',
-                'guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_on_a_filesystem',
-                'guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_to_postgresql',
-                'guides/setup/configuring_metadata_stores/how_to_configure_a_validation_result_store_in_amazon_s3',
-                'guides/setup/configuring_metadata_stores/how_to_configure_a_validation_result_store_in_azure_blob_storage',
-                'guides/setup/configuring_metadata_stores/how_to_configure_a_validation_result_store_in_gcs',
-                'guides/setup/configuring_metadata_stores/how_to_configure_a_validation_result_store_on_a_filesystem',
-                'guides/setup/configuring_metadata_stores/how_to_configure_a_validation_result_store_to_postgresql',
-                'guides/setup/configuring_metadata_stores/how_to_configure_a_metricsstore'
-              ]
-            },
-            {
-              type: 'category',
-              label: 'Configuring Data Docs',
-              items: [
-                'guides/setup/configuring_data_docs/how_to_host_and_share_data_docs_on_a_filesystem',
-                'guides/setup/configuring_data_docs/how_to_host_and_share_data_docs_on_azure_blob_storage',
-                'guides/setup/configuring_data_docs/how_to_host_and_share_data_docs_on_gcs',
-                'guides/setup/configuring_data_docs/how_to_host_and_share_data_docs_on_amazon_s3'
-              ]
-            }
-          ]
+    docs: [{
```

Comment:
`sidebars.js` has some merge conflicts. Did anything need to change here as a part of your updates? I thought it was just the content of the files, not the general structure?

Reply:
I removed the `/docs/guides/miscellaneous/how-to-add-comments-to-a-page-on-docs.greatexpectations.io` link which was not needed (see comment on the JIRA task)
I'll fix the merge conflict.

Reply:
Makes sense, thanks for updating Jira! I'll resolve this conversation.

Reply:
VS Code somehow did "format on save" so the whole file was indented :D I reverted in the latest commit.